Jenkinsfile

```diff
@@ -16,293 +16,345 @@ pipeline {
         booleanParam(name: 'vendor', defaultValue: true, description: 'Vendor')
         booleanParam(name: 'windowsRS1', defaultValue: false, description: 'Windows 2016 (RS1) Build/Test')
         booleanParam(name: 'windowsRS5', defaultValue: false, description: 'Windows 2019 (RS5) Build/Test')
-  }
-  stages {
-    stage('Build') {
-      parallel {
-        stage('unit') {
-          when {
-            beforeAgent true
-            expression { params.unit }
-          }
-          agent { label 'amd64 && ubuntu-1804 && overlay2' }
-          environment { DOCKER_BUILDKIT='1' }
+    }
+    environment {
+        DOCKER_BUILDKIT = '1'
```

Comment:
We ran into an issue where it was set locally but then the tests would consistently fail on s390x-ubuntu-1604.  Was it still failing?

Reply:
Hm.. didn't know that was the case; would be interesting to know _why_ (we can still override it for s390x/power)

Reply:
Yeah, not sure why docker build with buildkit is bonking out on 18.06...docker-ce 18.06.3 was the last s390x package version we released because the hardware maintenance has been difficult and almost no users. Perhaps we should start disabling the tests for s390x because we don't intend to ship docker-ce packages for s390x in the future.

Reply:
i see you set `environment { DOCKER_BUILDKIT = '0' }` in s390x and ppc64le steps. i think that's fine for now.